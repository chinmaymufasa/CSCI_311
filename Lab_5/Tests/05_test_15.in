2
97
4
5
6
2
47
4
5
6
2
190
4
5
6
2
173
4
5
6
2
140
4
5
6
-1
