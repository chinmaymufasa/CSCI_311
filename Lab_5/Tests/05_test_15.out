97 
97 
97 
97 47 
47 97 
47 97 
97 47 190 
47 97 190 
47 190 97 
97 47 190 173 
47 97 173 190 
47 173 190 97 
97 47 173 140 190 
47 97 140 173 190 
47 140 190 173 97 
